### PR TITLE
[Swift] Update for SILModule API change

### PR DESCRIPTION
**Pull request by slavapestov**

### CODE REVIEW

**Comment by slavapestov** [comment]
@swift-ci Please test

**Comment by slavapestov** [comment]
https://github.com/apple/swift/pull/26897
@swift-ci Please test

**Comment by slavapestov** [comment]
apple/swift#26897
@swift-ci Please test

File: include/lldb/Symbol/SwiftASTContext.h [hunk 1]
```diff
@@ -808,6 +813,7 @@ class SwiftASTContext : public TypeSystem {
   /// should be printed by Process::PrintWarningCantLoadSwift().
   std::vector<std::string> m_module_import_warnings;
   swift::ModuleDecl *m_scratch_module = nullptr;
+  swift::Lowering::TypeConverter *m_sil_types = nullptr;
```
Comment (jrose-apple) [inline include/lldb/Symbol/SwiftASTContext.h]: How does this get destroyed?

Comment (slavapestov) [reply include/lldb/Symbol/SwiftASTContext.h]: Good question.

**Comment by slavapestov** [comment]
@swift-ci Please test

**Comment by slavapestov** [comment]
apple/swift#26897
@swift-ci Please test

**Comment by slavapestov** [comment]
apple/swift#26897
@swift-ci Please test

**Pull request merged**